rfcs/text/0042/risk.yml

```diff
@@ -0,0 +1,168 @@
+      example: 25.0
+      description: >
+        The contribution of Category 3 to the overall risk score (`calculated_score`).
+
+        Category 3 includes inputs from Vulnerabilities.
+
+        Risk Categories logically group risk inputs from various domain use cases - e.g. Alert, Posture Management, Vulnerabilities, and others.
+    - name: category_3_count
+      level: extended
+      type: long
+      example: 4920
+      description: >
+        The number of risk input documents that contributed to the Category 3
+        score.
+
+        Risk Categories logically group risk inputs from various domain use cases - e.g. Alert, Posture Management, Vulnerabilities, and others.
+    - name: category_4_score
+      level: extended
+      type: float
+      example: 55.0
+      description: >
+        The contribution of Category 4 to the overall risk score (`calculated_score`).
+
+        Category 4 contains Entity Contexts.
+
+        Risk Categories logically group risk inputs from various domain use cases - e.g. Alert, Posture Management, Vulnerabilities, and others.
+    - name: category_4_count
+      level: extended
+      type: long
+      example: 1308
+      description: >
+        The number of risk input documents that contributed to the Category 4 score.
+
+        Risk Categories logically group risk inputs from various domain use cases - e.g. Alert, Posture Management, Vulnerabilities, and others.
+    - name: category_5_score
+      level: extended
+      type: float
+      example: 75.0
+      description: >
+        The contribution of Category 5 to the overall risk score (`calculated_score`).
+
+        Category 5 contains inputs from Anomalies.
+
+        Risk Categories logically group risk inputs from various domain use cases - e.g. Alert, Posture Management, Vulnerabilities, and others.
+    - name: category_5_count
+      level: extended
+      type: long
+      example: 14922
+      description: >
+        The number of risk input documents that contributed to the Category 5 score.
+
+        Risk Categories logically group risk inputs from various domain use cases - e.g. Alert, Posture Management, Vulnerabilities, and others.
+    - name: risk_inputs
+      level: extended
+      type: object
+      normalize: array
+      description: >
+        An array of the top risk input documents that contributed to the overall risk score.
+    - name: risk_inputs.id
+      level: extended
+      type: keyword
+      description: >
+        The identifier (_id) of the risk input document.
+    - name: risk_inputs.index
+      level: extended
+      type: keyword
+      description: >
+        The index (`_index`) of the risk input document.
+    - name: risk_inputs.risk_category
+      level: extended
+      type: keyword
+      example: category_1
+      description: >
+        The risk category of the risk input document.
+    - name: risk_inputs.risk_description
+      level: extended
+      type: keyword
```

Comment:
If you think having full-text indexing on the description may be useful, ECS does support defining multi-fields ([example](https://github.com/elastic/ecs/blob/main/schemas/file.yml#L85))

Reply:
Yep, good call 👍